neuland/calibration/R3BNeulandMapped2Cal.cxx

```diff
@@ -24,14 +24,16 @@
 #include "TClonesArray.h"
 #include "TH2F.h"
 #include "TMath.h"
+#include "mapping_neuland_trig.hh"
```

Comment:
I am wondering if this file depends on the experiment, if yes, it must be read from a parameter container. This kind of container was already created for other detectors like Fibers and TofD:
- https://github.com/R3BRootGroup/R3BRoot/blob/dev/tofd/pars/R3BTofDMappingPar.h
- https://github.com/R3BRootGroup/R3BRoot/blob/dev/fiber/pars/R3BFiberMappingPar.h

Reply:
There are also some examples of macros to convert the ucesb mapping files into r3broot parameter files:
https://github.com/R3BRootGroup/R3BParams_S515/blob/dev/macros/calibration/fibers/make_fiber_trig_mapping.C
https://github.com/R3BRootGroup/R3BParams_S515/blob/dev/macros/calibration/tofd/make_tofd_trig_mapping.C